### PR TITLE
Fix exports by removing hbs Prettier formatting

### LINKED ISSUE #363: Exporting of project is not working
After trying to export a project I got errors in console:

```
fetch.js:30          POST https://o114654.ingest.sentry.io/api/6391677/envelope/?sentry_key=5601cc9bb2594e769493ae68ac7d9bab&sentry_version=7&sentry_client=sentry.javascript.react%2F7.13.0 net::ERR_BLOCKED_BY_CLIENT
(anonymous) @ fetch.js:30
(anonymous) @ base.js:48
add @ promisebuffer.js:41
send @ base.js:63
_sendEnvelope @ baseclient.js:683
sendEvent @ baseclient.js:260
sendEvent @ client.js:95
(anonymous) @ baseclient.js:641
(anonymous) @ syncpromise.js:73
(anonymous) @ syncpromise.js:176
_executeHandlers @ syncpromise.js:170
(anonymous) @ syncpromise.js:91
a @ syncpromise.js:52
then @ syncpromise.js:63
_processEvent @ baseclient.js:609
_captureEvent @ baseclient.js:540
captureEvent @ baseclient.js:141
(anonymous) @ hub.js:181
_withClient @ hub.js:414
captureEvent @ hub.js:180
finish @ transaction.js:187
finish @ idletransaction.js:154
(anonymous) @ idletransaction.js:212
o @ helpers.js:85
setTimeout (async)
(anonymous) @ trycatch.js:107
_startIdleTimeout @ idletransaction.js:210
u @ idletransaction.js:97
f @ hubextensions.js:198
_createRouteTransaction @ browsertracing.js:164
(anonymous) @ browsertracing.js:100
(anonymous) @ router.js:52
p @ instrument.js:79
(anonymous) @ instrument.js:295
navigate @ history.js:87
onClick @ AccountList.tsx:57
Ve @ react-dom.production.min.js:52
Ge @ react-dom.production.min.js:52
(anonymous) @ react-dom.production.min.js:53
Di @ react-dom.production.min.js:100
ki @ react-dom.production.min.js:101
(anonymous) @ react-dom.production.min.js:113
Oe @ react-dom.production.min.js:292
(anonymous) @ react-dom.production.min.js:50
Pi @ react-dom.production.min.js:105
Qt @ react-dom.production.min.js:75
Zt @ react-dom.production.min.js:74
t.unstable_runWithPriority @ scheduler.production.min.js:18
Hr @ react-dom.production.min.js:122
Pe @ react-dom.production.min.js:292
Gt @ react-dom.production.min.js:73
o @ helpers.js:85
parser-babel.js:22 Uncaught (in promise) SyntaxError: Unexpected token (11:7) (at parser-babel.js:22:1038)
   9 |     // Set transaction signers const signers = [
  10 |     Alice,
> 11 |       ]
     |       ^
  12 |
  13 |
  14 |     // Generate addressMap from import statements
    at t.exports (parser-babel.js:22:1038)
    at t.exports (parser-babel.js:22:1271)
```

### PR DESCRIPTION
**Pull request by alse**
Closes: #363

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-playground/blob/master/CONTRIBUTING.md).
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: src/templates/js/config/README.md.hbs [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+
+## About Playground Export
+This project was generated from Playground living here:
+[{{projectLink}}]({{projectLink}})
+
+We have added some basics tests, which you can run using Node, Flow Emulator and Flow JS Testing Framework.
+
+## Installation
+### Node *(optional if you have it installed already)*
+Please follow instructions on NodeJS download page and install latest version of NodeJS software:
```
Comment (MrDSGC) [inline src/templates/js/config/README.md.hbs:17]: should we add something about node v17?

File: src/templates/js/partials/addSigners.hbs [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-  {{/each}}
-  ]
+{{#if signers ~}}
+    // Set transaction signers
+    const signers = [
+        {{#each signers ~}}
+            {{this}},
+        {{/each}}
+    ]
 {{/if}}
```
Comment (MrDSGC) [inline src/templates/js/partials/addSigners.hbs]: newline

File: src/templates/js/partials/quoteWrap.hbs [hunk 3]
```diff
@@ -1 +1 @@
-{{#if wrap}}"{{/if}}{{item}}{{#if wrap}}"{{/if}}
\ No newline at end of file
+{{#if wrap}}"{{/if}}{{item}}{{#if wrap}}"{{/if}}
```
Comment (MrDSGC) [inline src/templates/js/partials/quoteWrap.hbs:1]: newline

File: src/templates/js/testSuit.hbs [hunk 4]
```diff
@@ -1,29 +1,52 @@
...
+    {{#each transactionsUnitTests}}
+        {{{this}}}
+    {{/each}}
+})
+
+describe("Scripts", () => {
+    {{#each scriptsUnitTests}}
+        {{{this}}}
+    {{/each}}
 })
```
Comment (MrDSGC) [inline src/templates/js/testSuit.hbs]: newline

**Review by MrDSGC — APPROVED** [review]
1 liner fixes then lgtm

File: src/templates/js/config/README.md.hbs (hunk 1 above)
Comment (alse) [reply src/templates/js/config/README.md.hbs:17]: I added node 16 to the package.json engines

**Pull request merged**